gui.py

```diff
@@ -0,0 +1,60 @@
+import os
+import ntpath
+import tkinterdnd2
+
+from tkinter import *
+from tkinterdnd2 import *
+
+from page_separator import insert_suffix, main
+
+BG_COLOR = "#D4F1F4"
+
+def on_drop(event):
+    source_pdf.set(event.data)

```

Comment:
I'm getting braces around the directory strings. Although I'm not 100% sure after some research, it seems that when there are things like spaces in string, tkinter put braces around it (the string was probably first recognized as a list and it got "concatenated" when event.data was generated). I'm not sure if having braces as part of the input will interfere with page_separator or not, but it's visually ugly in the gui. But since the string must be a directory, maybe we can check if the first character if it is '{' and do event.data[1:-1] instead.

Reply:
```suggestion
    d = event.data
    d = d[1:-1] if d.startswith("{") and d.endswith("}") else d
    source_pdf.set(d)
```